### ISSUE TITLE
getMIDIValuesForElement() always returns -1 for the pitch

### ISSUE THREAD

**Issue by samuelbradshaw** [rism-digital/verovio#2339]
**Describe the problem**
According to https://github.com/rism-digital/verovio/issues/944 and https://github.com/rism-digital/verovio/issues/1413, the `vrvToolkit.getMIDIValuesForElement()` method should return the pitch for a note (also duration and time), given the ID of the note from the rendered SVG. Duration and time seem to be returning as expected; however, the pitch always comes back as -1.

I discovered this while working on my own project, but I was also able to reproduce it in the Verovio MEI Viewer.

**To Reproduce**
1. Load "Etude in F Minor" at https://www.verovio.org/mei-viewer.xhtml
2. Open the JavaScript console.
3. Type `vrvToolkit.getMIDIValuesForElement('d414233e1143');` in the console and press Enter. (d414233e1143 is the ID of a note in the fourth measure. Any note ID will work to reproduce the issue.)

**Expected behavior**
An object should be returned with a valid MIDI pitch.

**Actual behavior**
An object is returned with "-1" as the pitch.

**Verovio information**
 - Version of Verovio: 3.6.0-dev-6fe35f7 (also tested with 3.5.0 and 3.4.0 and had the same issue)
 - Binding: JavaScript

**Environment information (as appropriate)**
 - OS: macOS 11.5.1 Big Sur
 - Browser: Chrome and Safari

**Screenshots**
<img width="378" alt="Screen Shot 2021-08-10 at 6 56 13 PM" src="https://user-images.githubusercontent.com/7323052/128953583-689cc4ec-4020-4cea-a88f-bbefc6c52fcb.png">

**Comment by lpugin** [comment]
Fixed with a7daec8a9c5d19cb8744fa919a7b9fa0ee58dcbe
```js
vrvToolkit.getMIDIValuesForElement('d414233e1143')
```
```js
Object { duration: 125, pitch: 56, time: 5000 }
```

**Comment by samuelbradshaw** [comment]
Awesome! Thank you!
